packages/experimental/Drawer/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@fluentui-react-native/experimental-drawer",
+  "version": "0.0.1",
+  "description": "A Native Drawer component using the Fluent Design System. Currently only implemented on Android",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "clean": "fluentui-scripts clean",
+    "depcheck": "fluentui-scripts depcheck",
+    "just": "fluentui-scripts",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "dependencies": {
+    "@fluentui-react-native/adapters": ">=0.7.5 <1.0.0",
+    "@fluentui-react-native/experimental-text": ">=0.6.28 <1.0.0",
+    "@fluentui-react-native/component-cache": "^1.2.3",
+    "@fluentui-react-native/framework": "0.6.6",
+    "@fluentui-react-native/interactive-hooks": ">=0.10.54 <1.0.0",
+    "@fluentui-react-native/pressable": ">=0.7.30 <1.0.0",
+    "@fluentui-react-native/theme-tokens": "^0.9.2",
+    "@fluentui-react-native/tokens": ">=0.10.3 <1.0.0",
+    "@fluentui-react-native/use-styling": ">=0.7.2 <1.0.0",
+    "react-native-svg": "^12.1.1",
+    "tslib": "^1.13.0"
+  },
+  "devDependencies": {
+    "@fluentui-react-native/eslint-config-rules": "^0.1.1",
+    "@office-iss/react-native-win32": "^0.63.7",
+    "@types/react-native": "^0.63.0",
+    "@uifabricshared/build-native": "^0.1.1",
+    "react": "16.13.1",
+    "react-native": "^0.63.4"
+  },
+  "peerDependencies": {
+    "react": ">=16.13.1",
+    "react-native": ">=0.63.4"
```

Comment:
We've upgraded to RN 0.64, please update the RN versions you're listing in your dependencies. React has been bumped as well